index.js

```diff
@@ -52,7 +53,7 @@ function openTag(node) {
       value = css.trim();
     }
 
-    if (isVHook(value) && value.namespace) {
+    if (value instanceof softHook || value instanceof attrHook) {
```

Comment:
Guess it's too late, but stil.
@nthtran do you think it make sense to have `instanceof` check here?

Problem is that instances of hooks used in vdom-to-html and users' libs can be different, they will point to different constructors of SoftHook and AttrHook.